Formula/l/ldid-procursus.rb

```diff
@@ -1,14 +1,24 @@
 class LdidProcursus < Formula
   desc "Put real or fake signatures in a Mach-O binary"
   homepage "https://github.com/ProcursusTeam/ldid"
-  url "https://github.com/ProcursusTeam/ldid.git",
-     tag:      "v2.1.5-procursus7",
-     revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
-  version "2.1.5-procursus7"
   license "AGPL-3.0-or-later"
-  revision 1
+  revision 2
   head "https://github.com/ProcursusTeam/ldid.git", branch: "master"
 
+  stable do
+    version "2.1.5-procursus7"
+    url "https://github.com/ProcursusTeam/ldid.git",
+      tag:      "v2.1.5-procursus7",
+      revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
```

Comment:
```suggestion
    url "https://github.com/ProcursusTeam/ldid.git",
        tag:      "v2.1.5-procursus7",
        revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
```
Let's follow the style used by other formulae; thanks!